install/centos-installation-guide.md

```diff
@@ -7,54 +7,20 @@
 >
 > - If you are installing on a system that already has Clear Containers or `runv` installed,
 >   first read [the upgrading document](../Upgrading.md).
->
-> - This install guide is specific to integration with Docker.  If you want to use Kata
->   Containers through Kubernetes, see the "Developer Guide" [here](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#if-you-want-to-run-kata-containers-with-kubernetes).
-
-This step is only required in case Docker is not installed on the system.
```

Comment:
(I know you didn't change this, but...) s/in case/ for either 'in the case that' or 'if' ?

Reply:
Fixed.